modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -26,9 +26,9 @@ In the table, features are marked with the following statuses:
 |===
 |{gitops-shortname} | kam |Argo CD CLI |Helm |Kustomize |Argo CD |Argo Rollouts |Dex |RH SSO |Argo CD Agent |{ocp}
 
+|1.18.0 |NA |3.1.5  TP |3.18.4 GA |5.7.0 GA |3.1.4 GA |1.8.3 GA |2.43.0 GA |NA |0.2.1 TP |4.14-4.19
```

Comment:
```suggestion
|1.18.0 |NA |3.1.6  TP |3.18.4 GA |5.7.0 GA |3.1.6 GA |1.8.3 GA |2.43.0 GA |NA |0.4.1 TP |4.14, 4.16-4.19
```

Do we still need to mention kam?

cc: @svghadi

Reply:
Yes, let's remove KAM from the table. 
Thanks for pointing that out.

Reply:
Fixed, removed from the table.